src/testFixtures/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/resource/model/induction/PreviousQualificationsResponseBuilder.kt

```diff
@@ -1,17 +1,19 @@
 package uk.gov.justice.digital.hmpps.educationandworkplanapi.resource.model.induction
 
-import uk.gov.justice.digital.hmpps.educationandworkplanapi.resource.model.AchievedQualification
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.resource.model.AchievedQualificationResponse
 import uk.gov.justice.digital.hmpps.educationandworkplanapi.resource.model.EducationLevel
 import uk.gov.justice.digital.hmpps.educationandworkplanapi.resource.model.PreviousQualificationsResponse
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.resource.model.education.aValidAchievedQualificationResponse
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.resource.model.education.anotherValidAchievedQualificationResponse
 import java.time.OffsetDateTime
 import java.util.UUID
 
 fun aValidPreviousQualificationsResponse(
   reference: UUID = UUID.randomUUID(),
   educationLevel: EducationLevel = EducationLevel.SECONDARY_SCHOOL_TOOK_EXAMS,
-  qualifications: List<AchievedQualification> = listOf(
-    aValidAchievedQualification(),
-    anotherValidAchievedQualification(),
+  qualifications: List<AchievedQualificationResponse> = listOf(
```

Comment:
The type `PreviousQualificationsResponse` is now generated (generated from the swagger spec) with a `qualifications` property of a list of `AchievedQualificationResponse` rather than `AchievedQualification` as before